moveit_commander/test/python_moveit_commander.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+import unittest
+import numpy as np
+import rospy
+import rostest
+import os
+
+from moveit_commander import RobotCommander
+
+
+class PythonMoveitCommanderTest(unittest.TestCase):
+    PLANNING_GROUP = "manipulator"
+
+    @classmethod
+    def setUpClass(self):
+        self.commander = RobotCommander("robot_description")
+        self.group = self.commander.get_group(self.PLANNING_GROUP)
+
+    @classmethod
+    def tearDown(self):
+        pass
+
+    def check_target_setting(self, expect, *args):
```

Comment:
rename something like `test__group__set_joint_value_target` to make it easier to for future devs to understand what this test does